src/PowerShellEditorServices/Session/RemoteFileManager.cs

```diff
@@ -74,16 +74,22 @@ public class RemoteFileManager
 
             Register-EngineEvent -SourceIdentifier PSESRemoteSessionOpenFile {0}
 
-            if ((Test-Path -Path 'function:\global:PSEdit') -eq $false)
+            if ((Test-Path -Path 'function:\global:Open-EditorFile') -eq $false)
             {{
-                Set-Item -Path 'function:\global:PSEdit' -Value $PSEditFunction
+                Set-Item -Path 'function:\global:Open-EditorFile' -Value $PSEditFunction
+                Set-Alias psedit Open-EditorFile -Scope Global
             }}
         ";
 
         private const string RemovePSEditFunctionScript = @"
-            if ((Test-Path -Path 'function:\global:PSEdit') -eq $true)
+            if ((Test-Path -Path 'function:\global:Open-EditorFile') -eq $true)
             {
-                Remove-Item -Path 'function:\global:PSEdit' -Force
+                Remove-Item -Path 'function:\global:Open-EditorFile' -Force
+            }
+
+            if (Get-Alias psedit)
+            {
+                Remove-Alias psedit
```

Comment:
Also, I believe `Remove-Alias` was added for PS Core.  In my Win PS 5.1 that command doesn't exist.
```
remove-alias : The term 'remove-alias' is not recognized as the name of a cmdlet, function, script file, or operable
program. Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
```
Try `Remove-Item alias:\psedit` instead.  That should work across all versions of PS.

Reply:
Fixed!